app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
Understood, but:

* it doesn't hurt to guard
* helpers like the `is-array` helper would be discovered automatically (because of the dash) in Ember 1.12
* it makes us feel better :smiley_cat: